src/schematools/events/consumer.py

```diff
@@ -0,0 +1,121 @@
+"""Kafka consumer."""
+from __future__ import annotations
+
+import json
+import logging
+import os
+
+from confluent_kafka import Consumer, Message
+from sqlalchemy.engine import Connection
+
+from schematools.events.full import EventsProcessor
+from schematools.types import DatasetSchema
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def _to_env_name(name: str) -> str:
+    """Convert param name to env var name."""
+    return name.replace(".", "_").upper()
+
+
+def _fetch_consumer_params() -> dict:
```

Comment:
DSO APi uses a library `django-environ` for getting settings from the environment, though we seem to be using none of the Django specific features (if I recall correctly). Dataservice Airflow uses another library (`environs`). Here we use a custom solution. I think we can replace all these with the arguably much more elegant `pydantic` library and specifically this https://pydantic-docs.helpmanual.io/usage/settings/

Reply:
Though probably out of scope of this PR. Just wanted to bring `pydantic` to this group's attention.

Reply:
Yeah, good idea, `pydantic` rocks.